keras_cv/layers/preprocessing/README.md

```diff
@@ -0,0 +1,48 @@
+# Base Layers
+
+- BaseImageAugmentationLayer
+- VectorizedBaseImageAugmentationLayer
+- RandomAugmentationPipeline
+
+# Preprocessing Layers
+
+| Layer Name | Base Layer | Segmentation Masks | BBoxes |
+| :-- | :-- | :--: | :--: |
+| AugMix | BaseImageAugmentationLayer | ❌ | ✅ |
+| AutoContrast | VectorizedBaseImageAugmentationLayer | ✅ | ✅ |
+| ChannelShuffle | VectorizedBaseImageAugmentationLayer | ✅ | ✅ |
+| CutMix | BaseImageAugmentationLayer | ✅ | ❌ |
+| Equalization | BaseImageAugmentationLayer | ✅ | ✅ |
+| FourierMix | BaseImageAugmentationLayer | ❌ | ❌ |
+| Grayscale | VectorizedBaseImageAugmentationLayer | ✅ | ✅ |
+| GridMask | BaseImageAugmentationLayer | ✅ | ✅ |
+| JitteredResize | VectorizedBaseImageAugmentationLayer | ✅ | ✅ |
+| MixUp | BaseImageAugmentationLayer | ✅ | ✅ |
+| Mosaic | VectorizedBaseImageAugmentationLayer | ✅ | ✅ |
+| Posterization | BaseImageAugmentationLayer | ✅ | ✅ |
+| RandAugment | RandomAugmentationPipeline | ❌ | ❌ |
+| RandomApply | BaseImageAugmentationLayer | ❌ | ❌ |
```

Comment:
Meta layers like Randomapply/RandomChoice support all data types that their sub-layers support. Maybe a checkmark with an asterisk and an explanation below would be best for this one.